### PR TITLE
Fix #64: Test server: Endpoints for operation approvals

### LINKED ISSUE #64: Test server: Endpoints for operation approvals

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/OperationApproveInternalRequest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import lombok.Data;
+
+/**
+ * Request for approving an operation.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Data
+public class OperationApproveInternalRequest {
```
Comment (romanstrobl) [inline powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/OperationApproveInternalRequest.java:28]: The class looks dense, can we improve the formatting a bit?

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/response/CreateTokenResponse.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import lombok.Data;
+
+/**
+ * Response with created token.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Data
+public class CreateTokenResponse {
```
Comment (romanstrobl) [inline powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/response/CreateTokenResponse.java:28]: dtto

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/service/OperationsService.java [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    @SuppressWarnings("unchecked")
+    public OperationListResponse getOperations(GetOperationsRequest request) throws RemoteExecutionException, RestClientException, SignatureVerificationException, ActivationFailedException {
+        final JSONObject resultStatusObject = resultStatusUtil.getTestStatus(request.getActivationId());
+
+        final VerifyTokenStepModel model = new VerifyTokenStepModel();
+        model.setTokenId(request.getTokenId());
+        model.setTokenSecret(request.getTokenSecret());
+        model.setDryRun(true);
+        model.setHttpMethod("POST");
+        model.setUriString("--DUMMY");
```
Comment (romanstrobl) [inline powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/service/OperationsService.java:109]: The value gets overwritten two lines later.

File: powerauth-test-server/src/main/resources/application.properties [hunk 4]
```diff
@@ -15,6 +15,16 @@ spring.datasource.username=sa
 spring.datasource.password=
 spring.datasource.driver-class-name=org.h2.Driver
 
+# PostgreSQL configuration
+#spring.datasource.url=jdbc:postgresql://localhost:5432/powerauth
+#spring.datasource.username=powerauth
+#spring.datasource.password=
+#spring.datasource.driver-class-name=org.postgresql.Driver
+#spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```
Comment (romanstrobl) [inline powerauth-test-server/src/main/resources/application.properties:23]: We should probably use `PostgreSQL95Dialect` in the example, `PostgreSQLDialect` is deprecated.

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/CreateTokenRequest.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+import lombok.Data;
+
+/**
+ * Request for creating a token.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Data
+public class CreateTokenRequest {
```
Comment (romanstrobl) [inline powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/CreateTokenRequest.java:28]: Can we improve the formatting a bit? The class looks dense.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
Looks great! We should remove one `DUMMY` string, but the rest is without any issues.

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/OperationApproveInternalRequest.java (hunk 1 above)
Comment (petrdvorak) [reply powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/request/OperationApproveInternalRequest.java:28]: No, it's OK.

File: powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/response/CreateTokenResponse.java (hunk 2 above)
Comment (petrdvorak) [reply powerauth-test-server/src/main/java/com/wultra/security/powerauth/app/testserver/model/response/CreateTokenResponse.java:28]: No

**Review by romanstrobl — APPROVED** [review]
Thanks for the fixes.

**Pull request merged**